pallets/proposals/src/mock.rs

```diff
@@ -207,6 +208,10 @@ impl pallet_proposals::Config for Test {
     type RefundsPerBlock = RefundsPerBlock;
     type IsIdentityRequired = IsIdentityRequired;
     type MilestoneVotingWindow = MilestoneVotingWindow;
+    // TODO: all the mocks as they are dependant on orml-xtokens for testing
+    // We could do a mockrefundhandlerbasic without the bells and whistels to help also. which is what i will do.
```

Comment:
remove comments as the mock handler exists
Actually need to do a dummy implementation